lua/plugins/banhammer.lua

```diff
@@ -21,6 +21,33 @@ local function markup_tempban(chat_id, user_id, time_value)
 			{text = 'minutes', callback_data = ('tempban:ban:m:%s:%s'):format(user_id, chat_id)},
 			{text = 'hours', callback_data = ('tempban:ban:h:%s:%s'):format(user_id, chat_id)},
 			{text = 'days', callback_data = ('tempban:ban:d:%s:%s'):format(user_id, chat_id)},
+		},
+		{
+			{text = i18n('Cancel'), callback_data = ('tempban:cancel:%s'):format(user_id)},
+		}
+	}}
+
+	return markup
+end
+
+local function markup_mute(chat_id, user_id, time_value)
```

Comment:
Would've been better if you could reuse the keyboard for tempban instead of copy-pasting it's code.